thomaspoignant/go-feature-flag
# fix: CI - The assign/unassign GitHub Action is not working

## Description

- Problem:
The `takanome-dev/assign-issue-action` (currently used) only works properly when assigning issues, not when unassigning.

- The way it was resolved:
  - With assign assignees:
Use custom script to check comments containing `/assign-me` to specify assignees, add necessary labels and comments
  - With unassign assignees
Use `actions/stale@v9` to check for stale issues when specified conditions are met
Use custom script to unassign and delete specific labels

- How to test the change:
Change workflow schedule (`cron`), number of days before being marked as stale (`env.DAYS_UNTIL_STALE`) in `unassign-issue.yml`, and check

- **Note**:
In my previous PR, [this line](https://github.com/thomaspoignant/go-feature-flag/blob/main/exporter/fileexporter/exporter.go#L84) was not covered so I added a test case for it.

## Closes issue(s)

Resolve #2313 

## Checklist
